
From: Christoph Hellwig <hch@lst.de>

These had been officially deprecated since Rusty's module rewrite, but
never got the __deprecated marker.  The only remaining users are drm and
mtd, so we'll get some warnings for common builds.  But maybe that's the
only way to get the drm people to fix the mess :)

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/include/linux/module.h |   12 +++++++-----
 1 files changed, 7 insertions(+), 5 deletions(-)

diff -puN include/linux/module.h~mark-inter_module_-deprecated include/linux/module.h
--- 25/include/linux/module.h~mark-inter_module_-deprecated	2004-09-26 17:20:53.593268264 -0700
+++ 25-akpm/include/linux/module.h	2004-09-26 17:20:53.596267808 -0700
@@ -575,10 +575,12 @@ struct obsolete_modparm __parm_##var __a
 
 /* Use symbol_get and symbol_put instead.  You'll thank me. */
 #define HAVE_INTER_MODULE
-extern void inter_module_register(const char *, struct module *, const void *);
-extern void inter_module_unregister(const char *);
-extern const void *inter_module_get(const char *);
-extern const void *inter_module_get_request(const char *, const char *);
-extern void inter_module_put(const char *);
+extern void __deprecated inter_module_register(const char *,
+		struct module *, const void *);
+extern void __deprecated inter_module_unregister(const char *);
+extern const void * __deprecated inter_module_get(const char *);
+extern const void * __deprecated inter_module_get_request(const char *,
+		const char *);
+extern void __deprecated inter_module_put(const char *);
 
 #endif /* _LINUX_MODULE_H */
_
